controllers/openstackdataplanerole_controller.go

```diff
@@ -313,6 +325,44 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 	return ctrl.Result{}, nil
 }
 
+// ReconcileBaremetalSet Reconcile OpenStackBaremetalSet
+func (r *OpenStackDataPlaneRoleReconciler) ReconcileBaremetalSet(ctx context.Context, instance *dataplanev1beta1.OpenStackDataPlaneRole, helper *helper.Helper,
+) error {
+	baremetalSet := &baremetalv1.OpenStackBaremetalSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instance.Name,
+			Namespace: instance.Namespace,
+		},
+	}
+
+	helper.GetLogger().Info("Reconciling BaremetalSet")
```

Comment:
We can use `r.Log.Info()` here. The logger is initialized as part of the `OpenStackDataPlaneRoleReconciler` struct, so we can just use that instead of recalling `helper.GetLogger()`.